packages/@aws-cdk/cloudformation-diff/lib/diff-template.ts

```diff
@@ -184,3 +208,98 @@ function deepCopy(x: any): any {
 
   return x;
 }
+
+function filterFalsePositivies(diff: types.TemplateDiff, changeSet: CloudFormation.DescribeChangeSetOutput) {
+  const replacements = findResourceReplacements(changeSet);
+  diff.resources.forEachDifference((logicalId: string, change: types.ResourceDifference) => {
+    if (!replacements[logicalId] && !diff.resources.get(logicalId).otherChanges.DependsOn) {
```

Comment:
This needs a comment, not sure what's happening here.

Reply:
ahh good catch, this is a leftover